README.md

```diff
@@ -1,70 +1,188 @@
-# Getting Started with Create React App
+<div align="center"><h1>REACT CAPSTONE</h1>
+<img src="https://github.com/Atril33/country-details/blob/dev/Live%20Preview.png"></div>
+<!-- 
+HOW TO USE : 
+This is an example of how you may give instructions on setting up your project locally.
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+Modify this file to match your project and remove sections that don't apply.
 
-## Available Scripts
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+- Authors
+- Future Features- 
+ - Mobile version design
+- Desktop version design
+- Dynamic HTML for featured-speakers
 
```

Comment:
- [ ] Great work on making the live video 👍 but I guess you have missed the requirement which says that show an interesting piece of code that you are proud of 🤔 Kindly re-record the video and show an interesting piece of code from your project 😊